moped-editor/src/queries/project.js

```diff
@@ -199,45 +199,14 @@ export const UPSERT_PROJECT_PERSONNEL = gql`
       objects: $objects
       on_conflict: {
         constraint: moped_proj_personnel_project_id_user_id_role_id_key
-        update_columns: [project_id, user_id, role_id, status_id, notes]
+        update_columns: [project_id, user_id, role_id, is_deleted, notes]
       }
     ) {
       affected_rows
     }
   }
 `;
 
-export const UPDATE_PROJECT_PERSONNEL = gql`
```

Comment:
This query isn't used anywhere in the UI, and any updates we make to personnel records are handle with `UPSERT_PROJECT_PERSONNEL`. There might be a good reason to keep it so let me know, and I'll put it back.

Reply:
🙌